tests/drivers/gpio/gpio_basic_api/testcase.yaml

```diff
@@ -14,3 +14,17 @@ tests:
     platform_allow: nrf52840dk_nrf52840
     extra_args: "DTC_OVERLAY_FILE=boards/nrf52840dk_nrf52840.overlay;\
                  boards/nrf52840dk_nrf52840_sense_edge.overlay"
+
+  drivers.gpio.pcal6408a:
```

Comment:
I think a single test case is fine - `drivers.gpio.pcal64xxa`

Reply:
obsolete, removed the test